### DISCUSSION TITLE
Can't Push

### DISCUSSION THREAD

**Discussion by jhonyboy47** [community/community#17785] [GitHub Education]
I am trying to push after committing my changes, but I get this error. 

`
remote: Permission to education/GitHubGraduation-2022.git denied to jhonyboy47.
fatal: unable to access 'https://github.com/education/GitHubGraduation-2022.git/': The requested URL returned error: 403
`

Am I not suppoused to push?

I also noticed the wording of "Submitting your pull Request" I am confused to what that means.

**Comment by icorbrey** [comment]
You should fork the repository and make your changes to that. The button to fork the repo is in the top right of the screen, and once you make changes there should be an option to open a pull request back to the main repo
